### ISSUE TITLE
Grafana dashboard for PPMS

### ISSUE THREAD

**Issue by saneshark** [department-of-veterans-affairs/va.gov-team#18514]
This one will require collaboration with Facilities team. As part of conversations taking place with them, they should be able to accept a query parameter for VAOS and that query parameter would be used to create a new tag in their instrumentation such that we can build a grafana dashboard.

This issue is blocked until they can make these appropriate changes.

**Comment by laurenernest** [comment]
@saneshark Can you link the blocking ticket?

**Comment by ajmagdub** [comment]
Created first pass on PPMS Grafana dashboard: 
http://grafana.vfs.va.gov/d/-xWGWMEGz/vaos-ppms?orgId=1&from=now-15d&to=now

Waiting on Vets API changes to add error tracking panels.

**Comment by grossb1** [comment]
Found a way to gather total errors via the following query.  A new ticket will be created to track detailed error information once the data is made available.
sum without (instance, az)(increase(haproxy_backend_response_errors_total{proxy="ppms_back"}[10m]))
